spec/features/finance/payment_breakdown_spec.rb

```diff
@@ -351,21 +351,19 @@ def then_i_should_see_the_correct_payment_summary
   end
 
   def then_i_should_see_the_correct_output_fees
-    all(".finance-panel")[0] do
-      expect(page).to have_content("Output payments")
-      expect(page).to have_content(number_to_pounds(@jan_starts[:output_payments][0][:per_participant]))
-      expect(page).to have_content(@jan_starts[:output_payments][0][:participants])
-      expect(page).to have_content(number_to_pounds(@jan_starts[:output_payments][0][:subtotal]))
-    end
+    all(".finance-panel")[0]
```

Comment:
not sure what this line does and can be deleted? or is there an assertion that is exists?

Reply:
@asmega I thought you added it in? It was showing up in my linter as incorrect with the `do` block

Reply:
Now removed, didn't need them!